components/Link/index.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export interface Link {
+  theme?: {
+    colors?: object;
+  };
+  skin?: 'default' | 'dark';
+  children?: React.ReactNode[] | React.ReactNode;
+}
```

Comment:
The name of this file usually is index.d.ts and I think that is necessary convert this props to have native anchor element props too.

```suggestion
import React, { AnchorHTMLAttributes } from 'react';

export interface Link extends AnchorHTMLAttributes<HTMLAnchorElement> {
  theme?: {
    colors?: {};
  };
  skin?: 'default' | 'dark';
  children?: React.ReactNode[] | React.ReactNode;
}
```